### ISSUE TITLE
Missing character from slate theme color variable 'ring'

### ISSUE THREAD

**Issue by coppinger** [huntabyte/shadcn-svelte#315]
### Change Type

Correction

### Proposed Changes

I noticed that on my project, using the slate theme, that buttons weren't highlighting when focused, as they do in the examples.

Upon further inspection, I found the culprit:

```css
cssVars: {
    dark: {
        ...
        ring: "24 5.7% 82.9" /* missing a % at the end */
```

There's a missing '%' character at the end of the color string for the ring value—adding the symbol in fixes the issue.

#314

**Comment by huntabyte** [comment]
My guy! Thank you! @coppinger

**Comment by huntabyte** [comment]
Closed by #314
